pythonsd/templates/pythonsd/organizers.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'pythonsd/base.html' %}
+
+
+{% block title %}Organizers{% endblock title %}
+
+
+{% block main %}
+<div class="container mt-3">
+<h1>Organizers</h1>
+
+<p>If you would like to reach out, please contact the <a href="mailto:sandiegopython-organizers@googlegroups.com">Python SD Organizers</a>.</p>
+
+
+{% if organizers %}
+<div class="row row-cols-2 row-cols-md-4">
+  {% for organizer in organizers %}
+  <div class="col mb-4">
+    <div class="card">
+      <img src="{{ organizer.photo.url }}" class="card-img-top" alt="{{ organizer.name }}">
+      <div class="card-body">
+        <h5 class="card-title">{{ organizer.name }}</h5>
+        <p class="card-text">{{ organizer.description }}</p>
```

Comment:
What is `organizer.description` referencing?

Reply:
Heh. Nothing. I thought it might be a good idea but decided maybe it wasn't necessary. Do you have any thoughts there or should we cut it?

Reply:
I think it comes down to how much information is necessary here vs. what's displayed at the linked profiles.

A quick blurb about the organizer could be a nice touch, but where would it come from? Pulled from one of the profiles? Written by the organizer? How often would it be updated? 

Another option could be a start date. This could be displayed as something like "Organizer since YEAR" or "Contributing for _ years" (using timesince).

I'm not sure if any extra information is necessary since there will always be more info when clicking through to a profile. On the other hand, the card format definitely allows for more information at a glance.

Reply:
The blurb could come from the database but it's probably overkill. I'll just cut it for now and we can add one later if it feels useful.